getting-started/windows-calico/quickstart.md

```diff
@@ -191,9 +320,9 @@ The following steps install a Kubernetes cluster on a single Windows node, with
    - Downloads Kubernetes binaries.
    - Configures {{site.prodnameWindows}} and starts the Calico service.
 
-1. To skip any of the above tasks, see [Configure installation parameters](#configure-installation-parameters).
+   To skip any of the above tasks, see [Configure installation parameters](#configure-installation-parameters).
 
-1. Run install-calico-windows.ps1 for your datastore with parameters for your implementation.
+   Run install-calico-windows.ps1 for your datastore with parameters for your implementation.
    You do not need to pass a parameter if the default value of the parameter is correct for you cluster.
 
    **Kubernetes datastore (default)**
```

Comment:
For this note at end of table...it appears to apply only to "EtcdTlsSecretName". If yes, move the note to that description. If it will apply to upcoming parameters, put note at start of the table...so users do not miss it. 

Note: Use namespace kube-system instead of calico-system if your Calico installation is non operator-managed.

I also suggest rewrite so "non-operator" managed is not at end of sentence:

"Note: If you are not using operator-based installation, use namespace..."